x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -82,6 +83,7 @@ export const ActionTypeMenu = ({
         selectMessage: actionTypeModel ? actionTypeModel.selectMessage : '',
         actionType,
         name: actionType.name,
+        isExperimental: actionTypeModel.isExperimental,
```

Comment:
I don't see the technical preview badge on the add connector modal

Reply:
are you sure you're not in the edit view? it only shows on create

Reply:
<img width="863" alt="Screen Shot 2022-07-25 at 11 05 12 AM" src="https://user-images.githubusercontent.com/6935300/180834741-54bdaad9-dde1-4dcb-838a-9996e71515fe.png">

Reply:
There is a modal version of the "Add Connector" form that shows up when you add a new connector from the Security Solution rule creation page

https://user-images.githubusercontent.com/13104637/180836482-bf1f8367-3470-44ce-90cf-389a98e00e78.mp4

Reply:
whoops we're not supposed to be active in that view. i think i was confused about what `SecurityConnectorFeatureId` was for. im removing it now